chainer/functions/activation/crelu.py

```diff
@@ -0,0 +1,61 @@
+import numpy
+
+from chainer import cuda
+from chainer import function
+from chainer import utils
+from chainer.utils import type_check
+
+
+class CReLU(function.Function):
+
+    """Concatenated Rectified Linear Unit."""
+
+    def __init__(self, axis=1):
+        self.axis = axis
+
+    def check_type_forward(self, in_types):
+        type_check.expect(
+            in_types.size() == 1,
+            in_types[0].dtype == numpy.float32
+        )
```

Comment:
I'll support negative axis after https://github.com/pfnet/chainer/issues/1282 is fixed.